content/consortium/minutes/2019-05-15-minutes.md

```diff
@@ -0,0 +1,77 @@
+---
+date: "2019-05-15T00:00:00+00:00"
+title: "May 15, 2019 Advisory Board Meeting"
+tags: ["Open Force Field Consortium", "Advisory Board", "minutes"]
+categories: ["Advisory Board Meeting Minutes"]
+draft: false
+description: "Meeting minutes"
+weight: 10
+author: "Karmen Condic-Jurkic and Arjun Narayanan"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/consortium/) met on 15 May, 2019.
+The minutes are summarized as follows:
+
+### Open Forcefield Advisory Board Meeting Minutes – May 15, 2019
+
+2019/05/15 Advisory Board Meeting Minutes
+
+
+**Overview** of the [roadmap](https://openforcefield.org/science/downloads/roadmap/roadmap-graphic-may-2019-update.pdf) for the remaining time of Year 1 and the accomplishments made so far (J. Chodera).
+
+**Torsions update** (L.-P. Wang):
+
+- The selected set of molecules for initial torsion scanning contains 486 small and chemically diverse molecules provided by Roche (less than 30 heavy atoms)- [“Roche set”](https://github.com/openforcefield/open-forcefield-data/tree/master/Torsion-Drives/Roche-Reference-Compounds)- more fragments will be added to this set in the future;
+
+- A subset of 820 1D torsions was selected for initial scanning, but this set will be expanded to include more torsion and 2D scans, which are currently omitted due to computational cost;
+
+- Current level of theory used to compute torsion profiles is B3LYP-D3(BJ)/DZVP, but a subset of 15 torsions has been selected for further QM benchmarking (more details on datasets and benchmarking can be found in this [Science Update](https://openforcefield.org/science/updates/2019-05-16-condicj/));
+
+- [ForceBalance](https://github.com/leeping/forcebalance) is ready to compute torsion parameters.
+
+**Valence and impropers update** (D. Mobley):
+
+- Some initial refitting on of impropers on trivalent nitrogens using a small set of molecules with different substituents on nitrogen has been done based on SMIRNOFF99Frosst parameters, but more molecules will be added to this dataset (pulled from eMolecule with fingerprint addition);
+
+- Impropers are being scanned while keeping torsions fixed using [geomeTRIC](https://github.com/leeping/geomeTRIC)- simplified and independent fitting runs will be performed initially to test the framework, but ultimately, ForceBalance should be used for fitting with all data simultaneously;
+
+- Data generated for torsions might be used for an initial valence fitting round;
+
+- C. Bannan is working on porting [Chemper](https://github.com/MobleyLab/chemper) to AMBER (work funded by MolSSI) 
```

Comment:
@bannanc -- I think something may have been lost in live-note-taking-shorthand here, but I'm not up to date with your current efforts. Could you confirm this, or provide a better sentence? (sorry for the ping out of the blue)

Reply:
Here's an edit of what Karmen wrote below:
C. Bannan is working on using Chemper to port AMBER protein force field parameters into SMIRNOFF format.

Reply:
Thanks, Caitlin!

Reply:
Thanks both!